src/test/java/org/ossgang/commons/GcTests.java

```diff
@@ -17,9 +20,13 @@ private GcTests() {
     public static void forceGc() {
         for (int run = 0; run < 10; run++) {
             WeakReference<?> ref = new WeakReference<>(new Object());
-            while (ref.get() != null) {
+            Await.await(() -> {
```

Comment:
The previous implementation goes in a never ending loop on openj9-11 jdk for some reason.. anyway this way is a bit more verbose but we have some errors in case it tries to cleanup for too long